### ISSUE TITLE
Webpack 5 should ignore files in App_Resources

### ISSUE THREAD

**Issue by PeterStaev** [NativeScript/NativeScript#9299]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 8.0.1
 - Cross-platform modules: 8.0.0
 - Android Runtime: 8.0
 - iOS Runtime: 8.0
 - XCode Version: 12.4
 - Plugin(s):

**Describe the bug**
Updating a project to NS 8 / Webpack 5 and seems it is trying to load files from App_Resources. For example, i have a custom entitlements located in `App_Resources/iOS` and webpack fails prepare:
```
ERROR in ./app/App_Resources/iOS/app.entitlements 1:0
Module parse failed: Unexpected token (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> <?xml version="1.0" encoding="UTF-8"?>
| <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
| <plist version="1.0">
 @ ./app/ sync .(xml|js|ts|s?css)$ ./App_Resources/iOS/app.entitlements
 @ ./app/__@nativescript_webpack_virtual_entry_typescript__ 3:16-92
```

**To Reproduce**
Add app.entitlements file to `App_Resources/iOS` and run `ns prepare ios`

**Expected behavior**

**Sample project**

**Additional context**

**Comment by rigor789** [comment]
@PeterStaev can you share the sample project? `App_Resources` should be excluded by default

**Comment by rigor789** [comment]
Oh another thing, try moving `App_Resources` outside of the `app` folder, it's been at the root level by default for a while now, and it's generally safer to keep it at the root level. I may have a clue why it's being included, I'll check in a bit.

**Comment by PeterStaev** [comment]
@rigor789 I can confirm that moving the App_Resource outside the app folder makes the problem to disappear. I'm ok with this solution, so if you think there isn't any bug when using a different folder than the default one in `nativescript.config.ts` file you can close this.

**Comment by rigor789** [comment]
I still want to fix it if nested, since some projects may still have that, thanks for confirming.

**Comment by rigor789** [comment]
Fixed in `@nativescript/webpack@5.0.0-beta.4`
